frontend/src/pages/signup-page.tsx

```diff
@@ -77,13 +77,14 @@ function signin() {
 
           <div className="mb-2">
             <input
+              type='text'
               {...register('email')}
-              type="email"
+            
```

Comment:
Why removed this?

Reply:
This @Aswin2255 any explanation?

Reply:
![image](https://github.com/krishnaacharyaa/wanderlust/assets/69797465/4b8f06d3-ef92-49d1-881b-26271fe952ac)

Reply:
Okay the test is failing because of type=email, it is first validating the mail and then rest of the errors pop up once the error is fine, this is browser doing. ideally shouldn't be this way.. okay good we removed

>  so I have again given as type = "email"
This again gave, I didn't get it !

Reply:
we have not removed type="email" it is still there you can check the signup page. i have changed the test case to solve the above mentioned issue

Reply:
Super